d3b_utils/aws_bucket_contents.py

```diff
@@ -73,6 +77,10 @@ def fetch_bucket_obj_info(
         # ETag comes back with unnecessary quotation marks, so strip them
         object["ETag"] = object["ETag"].strip('"')
 
+        if full_filepath:
+            key = urlparse(object["Key"]).path
```

Comment:
Why the urlparse?

Reply:
I was trying to avoid hardcoding index values in or replacing `source` in the key. But it seems `urlparse` had unintended side effects so I removed it.